packages/typegen/src/typegen.ts

```diff
@@ -69,7 +76,8 @@ export async function main() {
 
   opts.disableOptionalPathParameters = argv.disableOptionalPathParameters ?? true;
 
-  const [imports, schemaTypes, operationTypings] = await generateTypesForDocument(argv._[0] as string, opts);
+  const [imports, schemaTypes, operationTypings, banner] = await generateTypesForDocument(argv._[0] as string, opts);
+  console.log(banner, '\n');
```

Comment:
We should probably not log anything to stdout when the -b option is not supplied

Reply:
Agreed, I've made the changes.